publish.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -e
+
+PACKAGE_NAME="bitmovin-player-ui"
+CI_BRANCH=$TRAVIS_BRANCH
+
+echo "INFO branch is set to ${CI_BRANCH}"
+
+CHANNEL=-1
+NPM_TAG=-1
+MAJOR=-1
+MINOR=-1
+POSTIFX=-1
+VERSION=-1
+
+DRY_RUN_CMD=
+if ($DRY_RUN); then
+    DRY_RUN_CMD="--dry-run"
+    echo "INFO performing a dry run"
+fi
```

Comment:
What is the dry run used for? Assuming it is used for testing purposes, do we really want to commit it?

Reply:
I would like to leave it there, so we can set the `DRY_RUN` environment variable via CI and look at the output in production the first time w/o actually running the command

Reply:
Ok, fair point!